### ISSUE TITLE
How to use selenium-jupiter with local driver

### ISSUE THREAD

**Issue by Bukama** [bonigarcia/selenium-jupiter#90]
After I tested selenium-jupiter on my local machine I wanted to use it at my work.

I've created the following, very simple Test, using Chrome, but my long term goal is to use the `browsers.json`.

The `@SetSystemProperty` comes from the [JUnit Pioneer](https://junit-pioneer.org/) project I'm maintainer of.
The `ChromeDriver` is put into a folder "drivers" beneath `src/test/resources` so it is on classpath.

```
@SetSystemProperty(key = "webdriver.chrome.driver", value = "./drivers/chromedriver.exe")
@ExtendWith(SeleniumJupiter.class)
public class InputPageTests {

	@Test
	void navigateToInputPageChrome(ChromeDriver driver) {

		driver.get("www.google.de");
	}
}  
```

When executing the test, it fails because some external call to `https://chromedriver.storage.googleapis.com:443` (see log) which seems not available (`Caused by: java.net.UnknownHostException: chromedriver.storage.googleapis.com`).
Most probably because we have virtual desktops behind a firewall, due security restraints.

So two questsions:
Why is this call made and how can I avoid it to use the extension with a local provided driver? I really love to use the extension, because it makes developing Selenium tests so much more enjoyabale, e.g. with automatic screenshot etc.

---

**Browser**: Chrome 81 (in this case)
**Selenium-Jupiter version**: 3.3.5

**Error log**
```
[INFO] Running tests.BaseTest

17:02:55.245 [main] DEBUG io.github.bonigarcia.wdm.cache.ResolutionCache - Resolution chrome=81 in cache (valid until 17:47:21 01/11/2020 MEZ)

17:02:55.260 [main] DEBUG io.github.bonigarcia.wdm.cache.ResolutionCache - Resolution chrome81=81.0.4044.138 in cache (valid until 16:47:21 02/11/2020 MEZ)

17:02:55.260 [main] INFO io.github.bonigarcia.wdm.WebDriverManager - Using chromedriver 81.0.4044.138 (resolved driver for Chrome 81)

17:02:55.307 [main] INFO io.github.bonigarcia.wdm.WebDriverManager - Reading https://chromedriver.storage.googleapis.com/ to seek chromedriver

17:02:55.323 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ex-00000001: preparing request execution

17:02:55.323 [main] DEBUG org.apache.hc.client5.http.protocol.RequestAddCookies - Cookie spec selected: strict

17:02:55.338 [main] DEBUG org.apache.hc.client5.http.protocol.RequestAuthCache - Auth cache not set in the context

17:02:55.338 [main] DEBUG org.apache.hc.client5.http.impl.classic.ProtocolExec - ex-00000001: target auth state: UNCHALLENGED

17:02:55.338 [main] DEBUG org.apache.hc.client5.http.impl.classic.ProtocolExec - ex-00000001: proxy auth state: UNCHALLENGED

17:02:55.338 [main] DEBUG org.apache.hc.client5.http.impl.classic.ConnectExec - ex-00000001: acquiring connection with route {s}->https://chromedriver.storage.googleapis.com:443

17:02:55.338 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ex-00000001: acquiring endpoint (3 MINUTES)

17:02:55.338 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ex-00000001: endpoint lease request (3 MINUTES) [route: {s}->https://chromedriver.storage.googleapis.com:443][total available: 0; route allocated: 0 of 5; total allocated: 0 of 25]

17:02:55.354 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ex-00000001: endpoint leased [route: {s}->https://chromedriver.storage.googleapis.com:443][total available: 0; route allocated: 1 of 5; total allocated: 1 of 25]

17:02:55.370 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ex-00000001: acquired ep-00000000

17:02:55.370 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ex-00000001: acquired endpoint ep-00000000

17:02:55.370 [main] DEBUG org.apache.hc.client5.http.impl.classic.ConnectExec - ex-00000001: opening connection {s}->https://chromedriver.storage.googleapis.com:443

17:02:55.370 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ep-00000000: connecting endpoint (30 SECONDS)

17:02:55.370 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ep-00000000: connecting endpoint to https://chromedriver.storage.googleapis.com:443 (30 SECONDS)

17:02:55.401 [main] DEBUG org.apache.hc.client5.http.impl.io.DefaultManagedHttpClientConnection - http-outgoing-0: close connection IMMEDIATE

17:02:55.401 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ep-00000000: endpoint closed

17:02:55.401 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ep-00000000: discarding endpoint

17:02:55.401 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ep-00000000: releasing endpoint

17:02:55.401 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ep-00000000: connection is not kept alive

17:02:55.401 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ep-00000000: connection released [route: {s}->https://chromedriver.storage.googleapis.com:443][total available: 0; route allocated: 0 of 5; total allocated: 0 of 25]

17:02:55.401 [main] WARN io.github.bonigarcia.wdm.WebDriverManager - There was an error managing chromedriver 81.0.4044.138 (java.net.UnknownHostException: chromedriver.storage.googleapis.com: chromedriver.storage.googleapis.com) ... trying again using latest driver stored in cache

17:02:55.448 [main] DEBUG io.github.bonigarcia.wdm.cache.ResolutionCache - Resolution chrome=81 in cache (valid until 17:47:21 01/11/2020 MEZ)

17:02:55.463 [main] DEBUG io.github.bonigarcia.wdm.cache.ResolutionCache - Resolution chrome81=81.0.4044.138 in cache (valid until 16:47:21 02/11/2020 MEZ)

17:02:55.463 [main] INFO io.github.bonigarcia.wdm.WebDriverManager - Using chromedriver 81.0.4044.138 (resolved driver for Chrome 81)

17:02:55.463 [main] INFO io.github.bonigarcia.wdm.WebDriverManager - Reading https://chromedriver.storage.googleapis.com/ to seek chromedriver

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ex-00000002: preparing request execution

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.protocol.RequestAddCookies - Cookie spec selected: strict

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.protocol.RequestAuthCache - Auth cache not set in the context

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.ProtocolExec - ex-00000002: target auth state: UNCHALLENGED

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.ProtocolExec - ex-00000002: proxy auth state: UNCHALLENGED

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.ConnectExec - ex-00000002: acquiring connection with route {s}->https://chromedriver.storage.googleapis.com:443

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ex-00000002: acquiring endpoint (3 MINUTES)

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ex-00000002: endpoint lease request (3 MINUTES) [route: {s}->https://chromedriver.storage.googleapis.com:443][total available: 0; route allocated: 0 of 5; total allocated: 0 of 25]

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ex-00000002: endpoint leased [route: {s}->https://chromedriver.storage.googleapis.com:443][total available: 0; route allocated: 1 of 5; total allocated: 1 of 25]

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ex-00000002: acquired ep-00000001

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ex-00000002: acquired endpoint ep-00000001

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.ConnectExec - ex-00000002: opening connection {s}->https://chromedriver.storage.googleapis.com:443

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ep-00000001: connecting endpoint (30 SECONDS)

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ep-00000001: connecting endpoint to https://chromedriver.storage.googleapis.com:443 (30 SECONDS)

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.io.DefaultManagedHttpClientConnection - http-outgoing-1: close connection IMMEDIATE

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ep-00000001: endpoint closed

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.classic.InternalHttpClient - ep-00000001: discarding endpoint

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ep-00000001: releasing endpoint

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ep-00000001: connection is not kept alive

17:02:55.463 [main] DEBUG org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager - ep-00000001: connection released [route: {s}->https://chromedriver.storage.googleapis.com:443][total available: 0; route allocated: 0 of 5; total allocated: 0 of 25]

17:02:55.463 [main] ERROR io.github.bonigarcia.wdm.WebDriverManager - There was an error managing chromedriver 81.0.4044.138 (java.net.UnknownHostException: chromedriver.storage.googleapis.com)

io.github.bonigarcia.wdm.config.WebDriverManagerException: java.net.UnknownHostException: chromedriver.storage.googleapis.com

	at io.github.bonigarcia.wdm.WebDriverManager.getDriversFromXml(WebDriverManager.java:873)

	at io.github.bonigarcia.wdm.managers.ChromeDriverManager.getDriverUrls(ChromeDriverManager.java:101)

	at io.github.bonigarcia.wdm.WebDriverManager.createUrlHandler(WebDriverManager.java:755)

	at io.github.bonigarcia.wdm.WebDriverManager.download(WebDriverManager.java:627)

	at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:549)

	at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:746)

	at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:555)

	at io.github.bonigarcia.wdm.WebDriverManager.setup(WebDriverManager.java:285)

	at io.github.bonigarcia.seljup.SeleniumExtension.runWebDriverManagerIfNeded(SeleniumExtension.java:263)

	at io.github.bonigarcia.seljup.SeleniumExtension.resolveParameter(SeleniumExtension.java:218)

	at org.junit.jupiter.engine.execution.ExecutableInvoker.resolveParameter(ExecutableInvoker.java:216)

	at org.junit.jupiter.engine.execution.ExecutableInvoker.resolveParameters(ExecutableInvoker.java:183)

	at org.junit.jupiter.engine.execution.ExecutableInvoker.resolveParameters(ExecutableInvoker.java:144)

	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:96)

	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6(TestMethodTestDescriptor.java:212)

	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor$$Lambda$253.0000000010EAA3A0.execute(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:208)

	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:137)

	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:71)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:135)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask$$Lambda$185.0000000010E0E960.execute(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask$$Lambda$184.0000000010AA49D0.invoke(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask$$Lambda$183.0000000010E0E500.execute(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)

	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService$$Lambda$189.0000000010DD83E0.accept(Unknown Source)

	at java.util.ArrayList.forEach(ArrayList.java:1268)

	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask$$Lambda$185.0000000010E0E960.execute(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask$$Lambda$184.0000000010AA49D0.invoke(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask$$Lambda$183.0000000010E0E500.execute(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)

	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService$$Lambda$189.0000000010DD83E0.accept(Unknown Source)

	at java.util.ArrayList.forEach(ArrayList.java:1268)

	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask$$Lambda$185.0000000010E0E960.execute(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask$$Lambda$184.0000000010AA49D0.invoke(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask$$Lambda$183.0000000010E0E500.execute(Unknown Source)

	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)

	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)

	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)

	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)

	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)

	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:248)

	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$5(DefaultLauncher.java:211)

	at org.junit.platform.launcher.core.DefaultLauncher$$Lambda$150.0000000010DD3070.accept(Unknown Source)

	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:226)

	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:199)

	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:132)

	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)

	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)

	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)

	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)

	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)

	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)

Caused by: java.net.UnknownHostException: chromedriver.storage.googleapis.com

	at java.net.InetAddress.getAllByName0(InetAddress.java:1418)

	at java.net.InetAddress.getAllByName(InetAddress.java:1322)

	at java.net.InetAddress.getAllByName(InetAddress.java:1245)

	at org.apache.hc.client5.http.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:43)

	at org.apache.hc.client5.http.impl.io.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:118)

	at org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:413)

	at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:164)

	at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:174)

	at org.apache.hc.client5.http.impl.classic.ConnectExec.execute(ConnectExec.java:135)

	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)

	at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)

	at org.apache.hc.client5.http.impl.classic.ProtocolExec.execute(ProtocolExec.java:169)

	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)

	at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)

	at org.apache.hc.client5.http.impl.classic.HttpRequestRetryExec.execute(HttpRequestRetryExec.java:93)

	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)

	at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)

	at org.apache.hc.client5.http.impl.classic.ContentCompressionExec.execute(ContentCompressionExec.java:128)

	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)

	at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)

	at org.apache.hc.client5.http.impl.classic.RedirectExec.execute(RedirectExec.java:116)

	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)

	at org.apache.hc.client5.http.impl.classic.InternalHttpClient.doExecute(InternalHttpClient.java:178)

	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:75)

	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:89)

	at io.github.bonigarcia.wdm.online.HttpClient.execute(HttpClient.java:138)

	at io.github.bonigarcia.wdm.WebDriverManager.getDriversFromXml(WebDriverManager.java:858)

	... 73 common frames omitted

17:02:55.479 [main] WARN io.github.bonigarcia.seljup.SeleniumExtension - Driver handler for context id [engine:junit-jupiter]/[class:tests.BaseTest]/[method:name(org.openqa.selenium.chrome.ChromeDriver)] not found

[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.078 s <<< FAILURE! - in tests.BaseTest

[ERROR] name{ChromeDriver}  Time elapsed: 1.063 s  <<< ERROR!

org.junit.jupiter.api.extension.ParameterResolutionException: Failed to resolve parameter [org.openqa.selenium.chrome.ChromeDriver arg0] in method [void tests.BaseTest.name(org.openqa.selenium.chrome.ChromeDriver)]: io.github.bonigarcia.wdm.config.WebDriverManagerException: java.net.UnknownHostException: chromedriver.storage.googleapis.com
Caused by: io.github.bonigarcia.wdm.config.WebDriverManagerException: io.github.bonigarcia.wdm.config.WebDriverManagerException: java.net.UnknownHostException: chromedriver.storage.googleapis.com
Caused by: io.github.bonigarcia.wdm.config.WebDriverManagerException: java.net.UnknownHostException: chromedriver.storage.googleapis.com
Caused by: java.net.UnknownHostException: chromedriver.storage.googleapis.com
```

**Comment by bonigarcia** [comment]
It should not be necessary to use `@SetSystemProperty`, since Selenium-Jupiter uses internally [WebDriverManager ](https://github.com/bonigarcia/webdrivermanager)to manage the drivers required by Selenium WebDriver (e.g. chromedriver for Chrome).

It seems your machine is not able to download chromedriver. This is usually caused by a network configuration problem, for example, if you are using a proxy to connect to the Internet (is this your case?). If so, you need to tune the configuration of WebDriverManager. Take a look at [this](https://bonigarcia.github.io/selenium-jupiter/#tuning-webdrivermanager) and [this](https://github.com/bonigarcia/webdrivermanager#webdrivermanager-api).

**Comment by Bukama** [comment]
Thank you for your response!
Yes, as written, we are on virtual desktops behind several firewalls with strict rules. So I was thinking about providing the driver by myself (we got one accepted by security department) and tried to tell selenium-jupiter to use that, which did not worked (at least the way I tried). I'll try to set up the proxy, using the [proxy related API/system properties](https://github.com/bonigarcia/webdrivermanager#webdrivermanager-api) or if this does not work to simulate this cache. 

If it's okay for you I'll response here, otherwise I can reach out to you on twitter or else if you want.
